andela/healthchecks-apollo
# #152588130 Exceed grace and timeout durations

#### What does this PR do

* User should be able to set timeouts and grace values for days exceeding 30

#### Description of the task to be completed

* _Currently_
A user can only set timeouts and grace values for checks for a maximum of 30 days.

* _Expected_
This period should be extended because it is possible to have cron jobs that run once every 30+ days

* _Acceptance criteria_
GIVEN that as a user I want to set up a new job
WHEN selecting a grace period for that job
THEN I should have the option of setting a grace period value of more than 30 days 

#### How should this manually be tested?

* You can run the application using this command:
>./manage.py runserver

* Log in and create a check, you should be able to use the slider to set a value greater than 30 days

#### Any background context you want to provide?

#### Pivotal Tracker story

* See [Feature #152588130](https://www.pivotaltracker.com/story/show/152588130)

#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2017-11-08 at 15 24 19" src="https://user-images.githubusercontent.com/17122597/32549351-1fa6ce1e-c49a-11e7-9ff2-494967a3e478.png">


#### Questions:
